types/crypto-js/index.d.ts

```diff
@@ -44,7 +44,7 @@ declare namespace CryptoJS {
 	interface LibWordArray {
 		sigBytes: number,
 		words: number[],
-        toString(encoder?: Encoder): string;
+		toString(encoder?: Encoder): string;
```

Comment:
please amend this commit to the previous one and forcepush. The history is better to be clean ;)

Reply:
@misak113 Done :)